ocaml/libs/xapi-stdext/lib/xapi-stdext-threads/ipq.mli

```diff
@@ -0,0 +1,23 @@
+type 'a event = {ev: 'a; time: Mtime.t}
+
```

Comment:
This could use some explanation what this module does as an overview plus a short explanation for each function.

Reply:
I wonder if this is a priority queue that we could find somewhere else.

Reply:
Added some documentation.

Probably there are something similar, for the moment I would just clean our laundry.

Reply:
I would say resolved